npc/re/merchants/3rd_trader.txt

```diff
@@ -113,7 +113,7 @@ job3_guil01,91,93,3	script	Rare Herb Collector	1_M_03,{
 
 // Point Merchants (Sorcerer) :: point_3rd_trader
 //============================================================
--	shop	Shard_Seller	HIDDEN_NPC,no,6360:200,6363:200,6361:200,6362:200
+-	shop	Shard_Seller	HIDDEN_NPC,no,6360:200,6363:200,6361:200,6362:200,1000291:-1,1000292:-1,1000293:-1,1000294:-1,1000295:-1
```

Comment:
Add the point merchant that you removed on dicastes01 below.

`dicastes01,207,200,6	duplicate(pss)	Points Merchant#dic	8_F_GIRL`

Reply:
It was duplicated already, i deleted the old version from shops.txt